app/controllers/mixins/ems_common.rb

```diff
@@ -276,6 +276,7 @@ def button
         when "storage_scan"                     then scanstorage
         when "storage_tag"                      then tag(Storage)
         when "physical_storage_new"             then javascript_redirect(:action => 'new', :controller => 'physical_storage', :storage_manager_id => block_storage_manager_id(params[:id]))
+        when "host_initiator_define"            then javascript_redirect(:action => 'new', :controller => 'host_initiator', :storage_manager_id => block_storage_manager_id(params[:id]))
```

Comment:
We name this operation "define" and not "new" to reflect that we are only creating a definition of a virtual entity inside the storage. 
Not sure if straying from the standard naming convention is worth it. Maybe it's better to stick with a standard name?

Reply:
I changed it to host_initiator_new and only kept the define in the button labeling,